examples/end-to-end-examples/support_chatbot/mcli_yamls/finetune/finetune_composer_codebase.yaml

```diff
@@ -0,0 +1,214 @@
+name: mpt-30b_chat-composer
+
+compute:
+  gpus: 64  # Number of GPUs to use
+
+  ## These configurations are optional
+  # cluster: r0z0 # Name of the cluster to use for this run
+  #gpu_type: h100_80gb # Type of GPU to use.
+
+integrations:
+# Clone and install the llm-foundry repo so we can run scripts from it
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  pip_install: -e .[gpu]
+  ssh_clone: false # Should be true if using a private repo
+  git_commit: 496b50bd588b1a7231fe54b05d70babb3620fc72
+- git_branch: dev
+  git_repo: mosaicml/composer
```

Comment:
why do we need composer dev?

Reply:
believe it was an old bug but i believe it was fixed